pkg/core/consensus/user/sortition.go

```diff
@@ -77,7 +77,7 @@ func (v VotingCommittee) MarshalJSON() ([]byte, error) {
 	return json.Marshal(data)
 }
 
-// createSortitionHash takes Seed value 'seed', round number 'round', step number 'step', and iteration number 'i',
+// createSortitionHash takes Seed value 'seed', round number 'round', step number 'step', and committee member number 'i',
```

Comment:
What about index `instead` of `number`?
```suggestion
// createSortitionHash takes Seed value 'seed', round number 'round', step number 'step', and committee member index 'i',
```

Reply:
I wasn't convinced by index, because it can be confused with the index of the member in the provisioner list,
but I guess conceptually it's correct to consider 'i' as the index in the committee that has to be "filled in" with a new member (i.e. the one we're extracting now).

Maybe we could say it explicitly. Like:
"..., and the index 'i' of the committee member to be extracted." ?

Reply:
Yes, let's go for the explicitness